server/controllers/cookInfo.js

```diff
@@ -0,0 +1,8 @@
+const { getCookInfo } = require('../database/getQueries/getCookInfo');
+
+exports.get = (req, res) => {
+  const { mealId } = req.params;
+  getCookInfo(mealId)
+    .then(row => res.send(row))
+    .catch(() => res.send('Could not find cook'));
```

Comment:
res.send should include res.status(500).

Reply:
like so: res.status(500).res.send('Could not find cook')